weave/index/index_sqlite.py

```diff
@@ -175,8 +178,9 @@ def to_pandas_df(self, max_rows=1000, **kwargs):
             [info[1] for info in
              self.cur.execute("PRAGMA table_info(pantry_index)").fetchall()]
         )
+        query = "SELECT * FROM pantry_index LIMIT ? OFFSET ?"
```

Comment:
Hmm. Does SQLlite have an inherent ordering, or do we need an order by part here?

Reply:
I can add an order by. Interestingly enough this worked for me first, then I had to order the SQL to get that to work. If you want, I can order by uuid here too to get similar behavior.

Reply:
Digging into it, we do need an order by here and the inherent order is not guaranteed. I'll throw that in.